tests/std/tests/P0896R4_views_drop_while_death/test.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <cassert>
+#include <cstddef>
+#include <ranges>
+#include <span>
+
+#include <test_death.hpp>
+using namespace std;
+
+[[maybe_unused]] constexpr auto lambda = [x = 42](int) { return x == 42; };

```

Comment:
Ah this is to ensure that it is not implicitly generated?

I observe that you did not comment this in the other tests ;)

Reply:
> I observe that you did not comment this in the other tests ;)

Fair enough, and nobody complained about those - I suppose this is fine to leave as-is. Or maybe we should comment all of them so I won't be the only person who knows that we need a non-default-constructible function object type for these tests because a default-initialized _`semiregular-box`_ contains a value-initialized object when the object type is default constructible. I.e., if this function was default-constructible, line 19 wouldn't trigger the empty check.